File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void triangle(int b);

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n <= 0);
    triangle(n);
}

void triangle(int b)
{
    for (int a = 0; a < b; a++)
    {
        for (int c = b; c > a + 1; c--)
        {
            printf(" ");
        }
        for (int c = 0; c <= a; c++)
        {
            printf("#");
        }
        printf("  ");
        for (int d = 0; d <= a; d++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

height = 0
# Condition to checks if the input is between 1 and 8
while height > 8 or height < 1:
    height = get_int("Height: ")
space = height
blocks = 0

for i in range(height):
    blocks += 1
    x = 0
    # Print Whitespaces
    while x < space:
        if x != space - 1:
            print(" ", end="")
        x += 1
    x = 0
    # Print Blocks
    while x < blocks:
        if x == blocks - 1:
            print("#")
        else:
            print("#", end="")
        x += 1
    space -= 1
